src/WordSearch/WordSearchSolver.ts

```diff
@@ -0,0 +1,98 @@
+import trie = require('trie-prefix-tree');
+import {Point} from './Point';
+import {WordSearchResult} from './WordSearchResult';
+
+export class WordSearchSolver {
+  // The grid of letters which makes up the wordsearch puzzle
+  private _matrix: string[][];
+  // Transpositions to apply when finding words
+  private _directions: number[][];
+  // Potential words to find in the letter grid
+  private _targets: any;  // tslint:disable-line

```

Comment:
I tend to prefer:

```
// tslint:disable-next-line:no-any
```

Rather than the same line, turn everything off approach.